food_science_room.py

```diff
@@ -0,0 +1,52 @@
+class Food_Science_Room:
+    def_init_(self,global_state,inventory):
+        self.global_state = global_state
+        self.inventory = inventory
+        self.set_state()
+        self.name = "food_science_room"
+        self.description = """You enter the food science. culinary utensils now litter the floor. You hear a beeping from the ovens."""
```

Comment:
Triple quotes not needed; use paren quote concatenation. See https://docs.python.org/3/tutorial/introduction.html#strings.